### PR TITLE
proto: update oracle module

### PR DESCRIPTION
**Pull request by marbar3778**
## Description

- update oracle module to work with protobuf

### CODE REVIEW

**Comment by marbar3778** [comment]
this will fail ci, it is generally ready for review.

File: x/oracle/module.go
Comment (Jedi2002) [inline x/oracle/module.go:121]: @juniuszhou  Is there a reason we were returning for export ? If so we might want to keep it that way ?

Comment (Jedi2002) [reply x/oracle/module.go:121]: Actually , I see the the Init genesis is setting  the values , so I guess exporting it makes senses

File: x/oracle/genesis.go
Comment (Jedi2002) [inline x/oracle/genesis.go:20]: Yeah , I think panic would be the correct thing to do

**Review by Jedi2002 — APPROVED** [review]
Awesome Job , Lgtm 
But i'd wait for either @juniuszhou or @ElliotFriedman  , to approve before merging

**Comment by marbar3778** [comment]
upgrade merged is ages behind a working branch Closing this and will try to get it working on the new branch.

**Pull request closed without merging**
